man/systemd-vmspawn.xml

```diff
@@ -385,6 +385,27 @@
           <xi:include href="version-info.xml" xpointer="v256"/>
           </listitem>
         </varlistentry>
+
+        <varlistentry>
+          <term><option>--pass-ssh-key=</option><replaceable>BOOL</replaceable></term>
+
+          <listitem><para>By default an SSH key is generated to allow <command>systemd-vmspawn</command> to open
+          a D-Bus connection to the VM's systemd bus. Setting this to "no" will disable SSH key generation.</para>
```

Comment:
should say that this key is transient and ephemeral, and what that means (i.e. that the keys are generated and valid for this invocation of the VM only, and typically not persisted)